src/org/folio/Constants.groovy

```diff
@@ -42,7 +42,8 @@ class Constants {
     static String FOLIO_HELM_REPO_URL = "https://folio-org.github.io/folio-helm"
     static String FOLIO_HELM_HOSTED_REPO_NAME = "helm-hosted"
     static String FOLIO_HELM_HOSTED_REPO_URL = "https://repository.folio.org/repository/helm-hosted/"
-    static String FOLIO_NPM_REPO_URL = "https://repository.folio.org/repository/npm-folioci/"
```

Comment:
Should we replace the constant in cypressTestsPipeline.groovy pipeline and test compability?